### DISCUSSION TITLE
EOS社区治理和发展的深度思考 (1) （草稿） Part.1  Thoughts in-deep on EOS community governance and development (Draft)

### DISCUSSION THREAD

**Discussion by cryptosoul-vioiv** [famland/website#57] [Show and tell]
### Part.1 Thoughts in-deep on EOS community governance and development (Draft)
看到BM回归EOS社区，并提出了社区治理方案，我很高兴。本来是准备具体探讨BM提出的治理方案。可写着写着，感觉问题比较复杂。即使我反馈的问题BM看清楚了，他也不一定能明白为什么我会这样思考。因此，我想从更深层次、更根本层面，对EOS，乃至区块链社区治理做一个梳理。在这样一个基础上，再来探讨EOS的社区治理，相信大家交流起来会更清晰。本文主要是站在一个EOS社区用户的角度，与BM进行的一种交流和探讨。当然，也可以将本文视为一封给BM的公开信。不当之处，欢迎大家批评、指正。谢谢！
I am very happy to see BM return to the EOS community and put forward a community governance plan. Originally, I was going to specifically discuss the governance plan proposed by BM. But as I look into this, I feel that the problem is more complicated that I thought. Even if BM sees the problems that I listed, he may not understand why I am thinking on this way. Therefore, I want to sort out EOS and even blockchain community governance from a deeper and more fundamental level. On such a basis, let’s discuss EOS community governance. I believe we will communicate more clearly. This article is mainly an exchange and communication with BM from the perspective of an EOS community user.  Certainly,  this article can also be regarded as an open letter to BM. You are welcome to criticize and correct any improprieties. Thanks!

### 1. Community governance needs a fundamental consensus

无论是探讨区块链的社区治理，还是探讨EOS的社区治理，都离不开一个大前提。那就是：“我们社区治理的根本目标和目的究竟是什么”。这个问题，解决的就是社区治理的根本共识。如果BM和社区用户没能在这个根本点上达成共识，社区治理是很难成功的。至少现在，大家的认知是有差异的。对于绝大多数社区用户来说，他们的目标和目的很简单，就是想赚钱。而对于BM来说，他想得更远、更深，是为了营造一个更公平、更美好的新社会。这种目标上的不一致，决定了社区治理的根本共识无法形成。
Whether it is discussing the community governance of the blockchain or the community governance of EOS, it is inseparable from a major premise. That is: "What is the fundamental goal and purpose of our community governance?" The solution to this problem is the fundamental consensus of community governance. If BM and community users fail to reach a consensus on this fundamental point, community governance will be difficult to succeed. At least for now, everyone’s cognition are different. For the vast majority of community users, their goal and purpose is simple, that is, they want to make money. For BM, he thinks farther and deeper in order to create a fairer and better new society. This inconsistency in goals determines that the fundamental consensus of community governance cannot be formed.

EOS用户认为。你BM根本不关心币价，只追求你的情怀。而BM认为，我做EOS为的就是情怀，没有情怀，币价也就失去了意义。反之，你们只要按照我的情怀和梦想之路坚持下去，币价一定会起来。而且，是健康持续的发展。但社区用户没这个耐心。他们认为，来EOS就是为了赚钱。赚了钱，诗和远方我们也愿意与你一起去追求。但如果现在赚不到钱，谈其它啥都没意义。事实上，双方都相信，情怀、梦想与赚钱、现实是可以统一的。问题在于：至今双方并没有在这一点上达成共识。如果连这个最根本共识都没形成，就去奢谈社区治理，无论愿望有多么好、规划多么好，最终都会不欢而散。
EOS users think. BM doesn't care about currency prices at all, only pursues your dreams. And BM believes that EOS mean dreams, without dreams, the currency price will lose its meaning. On the contrary, as long as you stick to my feelings and dreams, the currency price will definitely rise. Moreover, it is a healthy and sustainable development. But community users do not have this patience. They believe that they came to EOS to make money. After money is made, we are willing to pursue poetry and dreams together with you. But before money is made, dreams are worthless to discuss. Thus, both parties believe that feelings, dreams, making money, and reality can be unified. The problem is: so far both sides have not reached a consensus on this point. If even this most fundamental consensus has not been formed, then go to talk about community governance extravagantly. No matter how good the wishes and planning are, eventually, they will leave unhappy.

以目前BM提出的这个社区治理草案为例。就算一切顺利，全部按BM的设想通过了。社区成员在选举时，一定会首先选择那些对提升币价有想法、有迅速赚钱欲望的代表。在没有系统性战略布局的前提下，炒币、拉盘、提升币价，一定是大多数社区成员乐于看到的。这么一来，急功近利、杀鸡取卵在所难免。而BM设想的“基于所有人的生命，自由，财产和正义原则的社区”，显然是不可能支持这么干的。可只要社区决议一旦形成，BM也干预不了。EOS社区就极有可能继续朝着浮躁、短视，简单、粗暴的方向发展。
Take an example for the current community governance draft proposed by BM. Even if everything goes well, all passed according to BM's vision. When the community members are elected, they will first choose those representatives who have ideas about raising the price of the coin and desire to make money quickly. In the absence of a systematic strategic layout, most community members are willing to see speculation, pump, and increase the price of the coin. In this way, it is inevitable to rush for quick success and Kill chickens and get eggs. The "community based on the principles of life, freedom, property and justice for all" envisioned by BM is obviously impossible to support this. But as long as the community decision is formed, BM cannot intervene. The EOS community is very likely to continue to develop in an impetuous, short-sighted, simple, and crude direction.

而BM此时一定极其失望，你们这么目光短浅、不管不顾的，还是社区建设者吗。跟那些无知的“暴民”又有什么区别。这时的BM会再一次陷入两难处境中，离开不是，继续推动也不是。
And BM must be extremely disappointed at this time. Are you community builders so short-sighted and disregarded? What's the difference from those ignorant "mobs". At this time, BM will once again fall into a dilemma, it is not to leave or keeping push.

这个问题，其实是BM一再与社区决裂的硬伤所在。
This problem is actually why BM has repeatedly broken with the community.

现阶段，无论是EOS社区，还是其他区块链社区，要兼顾眼前利益和长远发展，是极其困难的。这也是为什么区块链新经济问世10多年了，至今在全球范围内都还找不到一个成功社区的原因。
At this stage, whether it is the EOS community or other blockchain communities, it is extremely difficult to balance immediate interests and long-term development. This is why the new blockchain economy has been around for more than 10 years, and so far there is still no successful community in the world.

认知被束缚了。这是所有区块链社区建设失败的最根本原因，也是EOS社区治理找不到根本共识，达不成根本共识的根本原因。
Cognition is fettered. This is the most fundamental reason for the failure of all blockchain communities, and it is also the fundamental reason why EOS community governance cannot find a fundamental consensus and cannot reach a fundamental consensus.

而认知一旦被解放，眼前利益与长远发展，EOS社区的赚钱诉求与BM的情怀和梦想，是可以完美融合的。
Once the cognition is liberated, the immediate benefits and long-term development, the earning demands of the EOS community and the feelings and dreams of BM can be perfectly integrated.

要做到这一点真的不难，难的是认知。认知突破了，实现就自然而然了。中国古代思想家王阳明的“知行合一”，讲的就是这个道理。
It is really not difficult to do this, but the hard part is cognition. When the cognition breaks through, the realization will come naturally. This is what the ancient Chinese thinker Wang Yangming's "the unity of knowledge and action" talked about.

本文会试图从对区块链新经济的认知出发，发掘出一条在EOS社区中，大家可以完美达成共识的新路径。
This article will try to start from the cognition of the new blockchain economy and discover a new path in the EOS community where everyone can reach a perfect consensus.

认知的难度是客观存在的，希望大家多给自己一点耐心和信心。
The difficulty of cognition exists objectively, and I hope you can give yourself more patience and confidence.

**Comment by happybig** [comment]
很不错
